kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -702,6 +730,47 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
 
     assert(exMsg.contains("Missing checkpoint at version 1"))
   }
+
+  test("One compaction") {
+    testWithCompactionsNoCheckpoint(
+      Seq(0, 1, 2, 3, 4, 5),
```

Comment:
could you please add named params here? it would make these cases a lot more readable

e.g. an example from above, that is very readable

```scala
    testWithSingularAndMultipartCheckpoint(
      deltaVersions = 15L until 25L,
      checkpointVersions = Seq(20),
      startCheckpoint = Optional.of(20),
      versionToLoad = Optional.of(22))
```